### PR TITLE
Some scrutinizer phpdoc fixes

### PR DESCRIPTION
**Pull request by rullzer**
Easy one :)

CC: @nickvergessen @MorrisJobke @icewind1991 @LukasReschke

### CODE REVIEW

**Comment by icewind1991** [comment]
:+1:

File: lib/public/IL10N.php [hunk 1]
```diff
@@ -76,10 +76,10 @@ public function n($text_singular, $text_plural, $count, $parameters = array());
 	/**
 	 * Localization
 	 * @param string $type Type of localization
-	 * @param int|string $data parameters for this localization
+	 * @param \DateTime|int|string $data parameters for this localization
 	 * @param array $options currently supports following options:
 	 * 			- 'width': handed into \Punic\Calendar::formatDate as second parameter
-	 * @return string|false
+	 * @return string|int|false
```
Comment (nickvergessen) [inline lib/public/IL10N.php]: both changes are wrong?

Comment (rullzer) [inline lib/public/IL10N.php]: ?

**Comment by rullzer** [comment]
Fix for failing dav tests in https://github.com/DeepDiver1975/CalDAVTester/pull/2

File: lib/public/IL10N.php (hunk 1 above)
Comment (nickvergessen) [inline lib/public/IL10N.php]: nevermind

**Comment by nickvergessen** [comment]
:+1:

**Comment by nickvergessen** [comment]
So can we merge this? I don't think the failure is related to this here

**Comment by rullzer** [comment]
Yes should be safe IMO

**Comment by LukasReschke** [comment]
> Fix for failing dav tests in DeepDiver1975/CalDAVTester#2

Fork and adjust test file? 😉

**Pull request merged**